num/quat/util.go

```diff
@@ -0,0 +1,21 @@
+// Copyright ©2018 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package quat
+
+func lift(v float64) Quat {
```

Comment:
I'm sure this is a good name, but why `lift`? I feel like there's some concept I'm missing.

Reply:
These are as existed in the the original CL, however, my understanding is that that conventional direction for the real basis vector is upwards, with the three imaginary bases orthogonal to that.